src/load_profile.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import src.conf_logger
+
+
+class LoadProfile:

```

Comment:
A class used for representing time series data. We might want to extend this with methods to add/subtract different load profiles.